gspread/spreadsheet.py

```diff
@@ -19,37 +22,40 @@
 class Spreadsheet:
     """The class that represents a spreadsheet."""
 
-    def __init__(self, http_client: HTTPClient, properties: Dict[str, Any]):
+    def __init__(self, http_client: HTTPClient, properties: Dict[str, Union[str, Any]]):
```

Comment:
what is the point of `Union[str, Any]`? Is this not basically the same as `Any`?

Reply:
the idea is telling mypy it could be `Any`type but also it could be a string, therefor when you assigne the result value to a variable that expects a string, mypy is happy :upside_down_face: